src/Whoops/Resources/js/whoops.base.js

```diff
@@ -50,13 +50,25 @@ Zepto(function($) {
 	  var clipEl = document.getElementById("copy-button");
 	  var clip = new ZeroClipboard( clipEl );
 	  var $clipEl = $(clipEl);
+	  var $clipHelpEl = $("#help-clipboard");
 
 	  // show the button, when swf could be loaded successfully from CDN
 	  clip.on("load", function() {
 		  $clipEl.show();
+		  $clipHelpEl.show();
 	  });
   }
-  
+
+  $(document).on('click', '.Whoops-hider', function () {
+	  var $container = $('.Whoops.container');
+	  if ($container.hasClass('hidden') === false) {
```

Comment:
My personal code style dictates expressive conditionals, I use full comparisons rather than the negation operator as a matter of practice and habit. I've changed it to use the ! operator, per your input.